samples/google/sheets/program.py

```diff
@@ -94,11 +81,11 @@ def _read_values(id, slack_target):
         data_type, formatted, unformatted = row
         text = "Row {0}: {1} = formatted `{2!r}`, unformatted `{3!r}`"
         text = text.format(i + 1, data_type, formatted, unformatted)
-        slack.chat_postMessage(channel=slack_target, text=text)
+        print(text)
```

Comment:
No need for the `text` helper var anymore, a single `print(f"...") is short enough, I think, and therefore better

Btw, IMO the `value` helper var in the functions below is still useful, in contrast (because setting is a little complicated)

Reply:
python forces to two lines, so I kept the `text` var